README.md

```diff
@@ -1 +1,278 @@
-# goat-guardian
+# Goat Guardian
+
+[![Build Status](https://secure.travis-ci.org/arow-oss/goat-guardian.svg)](http://travis-ci.org/arow-oss/goat-guardian)
+[![Hackage](https://img.shields.io/hackage/v/goat-guardian.svg)](https://hackage.haskell.org/package/goat-guardian)
+[![Stackage LTS](http://stackage.org/package/goat-guardian/badge/lts)](http://stackage.org/lts/package/goat-guardian)
+[![Stackage Nightly](http://stackage.org/package/goat-guardian/badge/nightly)](http://stackage.org/nightly/package/goat-guardian)
+
+Goat Guardian is a reverse-proxy authentication server.  You can run it in
+front of an existing web application.  It will handle all authentication,
+including OAuth and email-based login.
+
+Goat guardian is meant to make it easy to add support for authentication to
+your web application.  It works great for demos, prototypes, internal web
+applications, rapid development with a small team.  It even works well for
+larger applications and teams that want an easy way to implement multiple forms
+of authentication!
+
```

Comment:
It may be nice to emphasize that goat guardian can be used for any applications written in any languages/frameworks.

Reply:
I added the following line!

> Goat Guardian can be used in front of web applications written in any language/framework.